lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -137,9 +137,16 @@ class CatalogController < ApplicationController
     ##
     # Configuration for partials
     config.index.partials.insert(0, :index_breadcrumb)
-    
-    config.show.metadata_partials = [:summary_field, :access_field]
-    
+
+    config.show.metadata_partials = [
+      :summary_field,
+      :access_field,
+      :background_field,
+      :scope_and_arrangement_field,
+      :related_field,
+      :admin_info_field
+    ]
```

Comment:
As it stands, I don't see why you would add a field to the `catalog_controller_field_accessors` that would not result in an additional metadata section, but you have obviously thought through this particular feature more than I.